pkgs/games/colobot/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, cmake, boost, SDL2, SDL2_image, SDL2_ttf, libpng
+, glew, gettext, libsndfile, libvorbis, libogg, physfs, openal
+, xmlstarlet, doxygen, python3, callPackage }:
+
+let
+  colobot-data = callPackage ./data.nix {};
+in
+stdenv.mkDerivation rec {
+  pname = "colobot";
+  version = "0.1.12-alpha";
+
+  # Mayb require an update tp package colobot-data as well
+  # in file data.nix next to this one
```

Comment:
So I'd remove these and rewrite it upwards:

```suggestion
```

Reply:
What do you mean "upwards"?

Reply:
> What do you mean "upwards"?

Right above `version = "...";`.